content/posts/February-2022.md

```diff
@@ -25,3 +25,21 @@ weight: 64
   ([packit#1483](https://github.com/packit/packit/pull/1483))
   ([packit.dev#390](https://github.com/packit/packit.dev/pull/390))
 - `python3-setuptools_scm` is now available by default in the sandbox.
+
+## Week 8 (February 18th - February 28th)
+
+- All source-git-commands were updated to append a
+  `From-source-git-commit` or `From-dist-git-commit` _Git-trailer_
+  to the commit messages they create in dist-git or source-git,
+  in order to save the hash of the commits from which these
+  commits were created. This information is going to be used to
+  tell whether a source-git repository is in sync with the
+  corresponding dist-git repository.
```

Comment:
To be honest, I am not sure if we want/need these technical source-git changes in the blog posts. @TomasTomecek WDYT? When looking at this through the packit-user perspective, I would be really confused.

Maybe, we can have a dedicated source-git changelog or at least split the bullet points.

Reply:
That's a generic question, I know.

Reply:
agree, removed